src/trip.core.js

```diff
@@ -507,7 +507,7 @@ Trip.prototype = {
     this.setTripBlock(o);
     this.showTripBlock(o);
 
-    if (this.settings.enableAnimation) {
+    if (TripUtils.isSet(o.enableAnimation, this.settings.enableAnimation)) {
```

Comment:
About the proposed pull request... I did try to get a visually wrong behavior implementing the scenario you describe but it did not cause any visual impact.

The current state of the animation is either always true or false. The call to `showCurrentTrip()` only acts on the current trip by both removing and adding animation all the time. The transition itself causes the animation to occur. If the current trip is not animated (whether or not the previous one was) there will not be a transition from "not animated" -> "animated" so no animation.

With the proposed change, the animation-related classes might remain on the trip block although the animation is disabled. It has no visual impact. So in your example, the first tripBlock is animated and the second is not although it will still have the `animated` and animation name class (e.g. `tada`).